src/data/symbol/symbol.tsv

```diff
@@ -90,35 +92,44 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	〘〙	() 「」 かっこ こうかっこ	二重甲括弧
 記号	〚〛	() 「」 かっこ かくかっこ	二重角括弧
 記号	｛｝	() 「」 かっこ なみかっこ	波括弧
-記号	〈〉	() 「」 かっこ やまかっこ	山括弧
-記号	《》	() 「」 かっこ やまかっこ	二重山括弧
-記号	«»	() 「」 かっこ やまかっこ ぎゅめ	二重山括弧・ギュメ
-記号	‹›	() 「」 かっこ やまかっこ	一重山括弧
+記号	〈〉	() 「」 <> かっこ やまかっこ	山括弧
+記号	《》	() 「」 <<>> かっこ やまかっこ	二重山括弧
+記号	«»	() 「」 <<>> かっこ やまかっこ ぎゅめ	二重山括弧・ギュメ
+記号	‹›	() 「」 <> かっこ やまかっこ	一重山括弧
 記号	「」	() 「」 かっこ かぎかっこ	かぎ括弧
 記号	『』	() 「」 かっこ かぎかっこ	二重かぎ括弧
 記号	【】	() 「」 かっこ 	すみつき括弧
-記号	‘’	() 「」 かっこ	シングル引用符
-記号	“”	() 「」 かっこ	ダブル引用符
-記号	〝〟	() 「」 かっこ つめかっこ だぶるみにゅーと ののかぎ	爪括弧
+記号	‘’	() 「」 "" '' かっこ	シングル引用符
+記号	“”	() 「」 "" '' かっこ	ダブル引用符
+記号	〝〟	() 「」 "" '' かっこ つめかっこ だぶるみにゅーと ののかぎ	爪括弧
 記号	＋	ぷらす たす すうがく + ＋	加算記号		SYMBOL GENERAL		U+FF0B FULLWIDTH PLUS SIGN
 記号	−	まいなす ひく はいふん すうがく - ー	減算記号 ハイフン		SYMBOL GENERAL		U+2212 MINUS SIGN
-記号	±	ぷらすまいなす すうがく きごう	正または負記号		SYMBOL
+記号	±	ぷらすまいなす ぷらまい +- すうがく きごう	正または負記号		SYMBOL
+記号	∓	まいなすぷらす -+ ー＋ すうがく きごう	負または正記号		SYMBOL
 記号	×	かける すうがく * ばつ ぺけ きごう	乗算記号		SYMBOL
 記号	÷	わる すうがく / ・ きごう	除算記号		SYMBOL
 記号	＝	いこーる とうごう すうがく =	等号		SYMBOL GENERAL
-記号	≠	ふとうごう のっといこーる すうがく = きごう とうごう	等号否定		SYMBOL
+記号	≠	ふとうごう のっといこーる すうがく = != <> きごう とうごう	等号否定		SYMBOL
 記号	＜	ふとうごう しょうなり すうがく <	不等号		SYMBOL GENERAL
 記号	＞	ふとうごう だいなり すうがく >	不等号		SYMBOL GENERAL
-記号	≦	ふとうごう しょうなりいこーる すうがく < きごう	不等号		SYMBOL
-記号	≧	ふとうごう だいなりいこーる すうがく > きごう	不等号		SYMBOL
+記号	≦	ふとうごう しょうなりいこーる すうがく < <= きごう	不等号		SYMBOL
+記号	≧	ふとうごう だいなりいこーる すうがく > >= きごう	不等号		SYMBOL
+記号	≤	ふとうごう しょうなりいこーる すうがく < <= きごう	不等号	海外	SYMBOL
+記号	≥	ふとうごう だいなりいこーる すうがく > >= きごう	不等号	海外	SYMBOL
+記号	⩽	ふとうごう しょうなりいこーる すうがく < <= きごう	不等号	海外	SYMBOL
+記号	⩾	ふとうごう だいなりいこーる すうがく > >= きごう	不等号	海外	SYMBOL
```

Comment:
I guess these 海外 symbols are not so popular, but confusing with ≦ and ≧.
How about removing them?

Reply:
We can frequently find them in higher education even in Japan, I think.
"大学" instead of "海外" might be better.
Do you still think they're unnecessary?

Reply:
- https://detail.chiebukuro.yahoo.co.jp/qa/question_detail/q13121735538
- https://manabitimes.jp/math/1313

Reply:
> Do you still think they're unnecessary?

I don't say they are unnecessary, but given that those we often use these characters can easily add them into user dictionary, I'd say the urgency is not that high. Actually, I'm more interested in how we can improve our candidate window UX as we keep increasing number of candidate, but it'd be a different story.

Reply:
> (*)

I guessed it. I'll remove "きごう" from the following symbols because they'll not be wanted by those who try to input symbols by "きごう".

- the above 4 inequality symbols
- mathematical dots

I don't know whether we should do the same for:

- ∓
- en dash
- (em dash) (should be always used instead of U+2015 if Windows didn't exist)

Reply:
> the above 4 inequality symbols

For those 4 inequality symbols, I'd suggest not including into this PR at least until we come up with a better description. For people who have never used those characters, I guess "大学", "大学・海外" and "海外" are all hard to understand what the description tried to explain. Let's revisit them later, e.g. 1 year later for example. Again, I don't think adding them are not that urgent.

Reply:
Removed it as you wish.
Also removed "きごう" from mathematical dots because they're not in JIS X 0213.
I found both dashes and ∓ are all in JIS X 0213.